### ISSUE TITLE
Scale up or create new variants from a generated image, like midjourney?

### ISSUE THREAD

**Issue by biocyberman** [emacs-openai/dall-e#2]
Thanks for very interesting package. The title said it all. Look forward to this feature :)

**Comment by jcs090218** [comment]
I agree; this is an interesting feature. 😋 However, I will be busy for the next couple of months. 😓 I will try to come back to this as soon as I have time!

Thank you for opening this feature request! ❤️ Anyways, PRs are welcome!

**Comment by biocyberman** [comment]
I think the facility is there already, it just needs some function to take the image , submit it with new output dimension requirement. My elisp is a bit rusty so it will take time to do it. So I will let you take lead :)
